src/java.base/share/native/libjava/System.c

```diff
@@ -146,20 +146,19 @@ Java_jdk_internal_util_SystemProps_00024Raw_platformProperties(JNIEnv *env, jcla
     PUTPROP(propArray, _path_separator_NDX, sprops->path_separator);
     PUTPROP(propArray, _line_separator_NDX, sprops->line_separator);
 
+    /* basic encoding properties, always non-NULL */
 #ifdef MACOSX
     /*
      * Since sun_jnu_encoding is now hard-coded to UTF-8 on Mac, we don't
      * want to use it to overwrite file.encoding
```

Comment:
This `file.encoding` can also be renamed. Should have been done with JEP 400

Reply:
Updated, thanks.